4001-5000/LIT4723TarikaWaldaAmid.xml

```diff
@@ -102,7 +103,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <ptr target="bm:Kropp2016Zekra"/><citedRange unit="page">1-12</citedRange>
                     </bibl>
                     <bibl>
-                        <ptr target="bm:Kropp1986ArabÄthÜbersetzungstechnik"/>
+                        <ptr target="bm:Kropp1986ArabAethÜbersetzungstechnik"/>
```

Comment:
there is still an Umlaut left here. It has been corrected in Zotero but not here.
No umlauts in Ids, links, filenames, foldernames please

Reply:
Yes. The branch has been already deleted. I think, I need to create a new branch to solve the problem.

Reply:
I corrected this in the file before merging this branch